src/main/java/com/auth0/jwk/Jwk.java

```diff
@@ -93,7 +122,7 @@ public String getUsage() {
     }
 
     @SuppressWarnings("WeakerAccess")
-    public String getOperations() {
+    public List<String> getOperations() {
```

Comment:
Ahh. This breaking change can't be avoided, right? 😞  Unless we add a different method to get them as a List<String>. What is your opinion on that? Because breaking is breaking even if it's 1 or 10 changes.

Reply:
We could name the new method something like "getOperationsAsList()"  which returns the List as is, then rework the existing method to return a String value representation of the List delimited by commas (if there are multiple entries) using a StringBuilder/for loop (since we're working with Java 7). It feels a little expensive but should keep us from breaking - let me know what you think.

Reply:
Yes, that could work. Ok. Lets add the `getOperationsAsList()` method and lets make the current one iterate through each item and prepare the _csv_. Note some edge-cases for this refactor:
- If the `key_ops` key wasn't present on the map when the "from map" static method was called, the List value will be null. This method will return a null String as well.
- If the `key_ops` key was present but empty or null, the list will be instantiated with a null or empty String (list.size() == 1). This probably won't happen a lot as whoever created the JWK will ensure the value is valid and non-empty, but anyway, I think in this case the method should return a null String as well.
- We should strip the commas from the end of the value. i.e.:
    - "one" instead of "one,"
    - "one,two" instead of "one,two,"